code/game/objects/structures/beds_chairs/chair.dm

```diff
@@ -62,20 +62,22 @@
 		qdel(src)
 
 /obj/structure/chair/attack_tk(mob/user)
-	if(buckled_mob)
+	if(buckled_mobs.len)
 		..()
 	else
 		rotate()
 	return
 
 /obj/structure/chair/proc/handle_rotation(direction)
-	if(buckled_mob)
-		buckled_mob.buckled = null //Temporary, so Move() succeeds.
-		if(!direction || !buckled_mob.Move(get_step(src, direction), direction))
-			buckled_mob.buckled = src
-			dir = buckled_mob.dir
-			return 0
-		buckled_mob.buckled = src //Restoring
+	if(buckled_mobs.len)
+		for(var/m in buckled_mobs)
+			var/mob/living/buckled_mob = m
+			buckled_mob.buckled = null //Temporary, so Move() succeeds.
+			if(!direction || !buckled_mob.Move(get_step(src, direction), direction))
+				buckled_mob.buckled = src
+				dir = buckled_mob.dir
+				return 0
+			buckled_mob.buckled = src //Restoring
```

Comment:
Unrelated to this PR, but the buckled trick here is no longer needed, it's been removed for the janicart for example.